src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static Random random = new Random();
+    protected static Circle circle = new Circle(random.nextInt(), ColorSupplier.getRandomColor());
+    protected static IsoscelesTrapezoid isoscelesTrapezoid
+            = new IsoscelesTrapezoid(random.nextInt(),
+            random.nextInt(), random.nextInt());
+    protected static Rectangle rectangle = new Rectangle(random.nextInt(),
+            random.nextInt());
+    protected static RightTriangle rightTriangle = new RightTriangle(random.nextInt(),
+            random.nextInt());
+    protected static Square square = new Square(random.nextInt());
```

Comment:
This action should be performed in methods.
no need to create figures as class variables
```suggestion
```